common/src/IO/TextureCollectionLoader.cpp

```diff
@@ -32,24 +33,38 @@
 
 namespace TrenchBroom {
     namespace IO {
-        TextureCollectionLoader::TextureCollectionLoader(Logger& logger) :
-        m_logger(logger) {}
+        TextureCollectionLoader::TextureCollectionLoader(Logger& logger, const std::vector<std::string>& exclusions) :
```

Comment:
It would be good to have a few tests that specifically check whether the exclusion works as expected. Maybe you can use one of the WAD files in the test fixtures for this.